src/coffee.js

```diff
@@ -51,16 +52,10 @@ const coffee = [
     acidity: "sweet",
     url: "https://impactroasters.dk/pages/about-coffe-coffee-guide",
     image: BezaImage,
-    characteristics: ["intense sweetness", "spicy", "forest berry aroma"],
+    characteristics: ["intense sweetness", "spicy", "forest berry"],
     processing: ["sundried"],
     roastLevels: ["dark", "light"],
-    preparation: [
-      "espresso",
-      "cafe latte",
-      "cappuccino",
-      "cortado",
-      "flat white",
-    ],
+    preparation: ["aeropress", "chemex and hario v60"],
```

Comment:
Let's capitalise these for all coffee:
```suggestion
    preparation: ["Aeropress", "Chemex and Hario v60"],
```